File: AMT-Services/src/main/java/am/infrastructure/data/enums/Roles.java
package am.infrastructure.data.enums;

/**
 * Created by mohamed.elewa on 5/18/2016.
 */
public enum Roles {
    ADMIN("Ad", "Admin"),
    STUDENT("St", "Student"),
    TUTOR("Tu", "Tutor");

    private String role;
    private String description;

    Roles(String role, String description){
        this.role = role;
        this.description = description;
    }
    public String description(){return description;}
    public String role(){return role;}

    @Override public String toString() { return description; }
}

File: Web-App/src/app/amt/util/constants/lookups/Roles.ts
import {Role} from "../../dto/lookup/Role";
/**
 * Created by ahmed.motair on 12/31/2017.
 */

export class Roles{
    static OWNER: Role = new Role('Ow', 'Owner');
    static ADMIN: Role = new Role('Ad', 'Admin');
    static TUTOR: Role = new Role('Tu', 'Tutor');
    static STUDENT: Role = new Role('St', 'Student');
}